### ISSUE TITLE
Make the Github Wiki Pull Requestable

### ISSUE THREAD

**Issue by TomKaltz** [CasparCG/server#889]
http://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html

**Comment by TomKaltz** [comment]
Should we do something like this or just allow certain contributors direct write access?

**Comment by ronag** [comment]
@dotarmin ?

**Comment by dotarmin** [comment]
> Should we do something like this or just allow certain contributors direct write access?

@TomKaltz I have talked with @silid today and we agreed on enabling pull requests for the Wiki according to the link you provided. What do you think? I personally think it's a good idea to at least try it and evaluate it and from there see how to proceed. 

Best regards,
Armin

**Comment by TomKaltz** [comment]
Yeah I think we should do that workflow.  Even the two-way syncing.  Not sure why it's not a native Github option but it seems like the way to go.  We need to make it as easy as possible for anyone in the community to contribute to documentation and allow moderators to review and approve those contributions.

**Comment by dotarmin** [comment]
Perfect! I couldn't agree more. I'll try to fix this asap.

**Comment by dotarmin** [comment]
I have thought about this again and think we should have a separate repository housing wiki for all projects. And then we can open up for contributors in a more standardised way.

Thoughts?

**Comment by TomKaltz** [comment]
We should house a seprate Repo for each Wiki we want to maintain and link/sync it to the project wiki just as described in the link above.

**Comment by dotarmin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [GitHub Wiki - Make pull request work](https://trello.com/c/EyQBy9yC/35-github-wiki-make-pull-request-work)
